package.json

```diff
@@ -47,6 +47,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.8.0",
+    "@sap-theming/theming-base-content": "^11.1.16",
```

Comment:
This would ideally come from fundamental-styles so we don't need an extra dependency.